src/cli/backup/teams.go

```diff
@@ -217,14 +254,55 @@ func validateTeamBackupCreateFlags(teams []string) error {
 		)
 	}
 
-	// TODO(meain)
-	// for _, d := range cats {
-	// 	if d != dataLibraries {
-	// 		return clues.New(
-	// 			d + " is an unrecognized data type; only  " + dataLibraries + " is supported"
-	// 		)
-	// 	}
-	// }
+	msg := fmt.Sprintf(
+		" is an unrecognized data type; only %s and %s are supported",
+		dataLibraries, dataMessages)
+
+	allowedCats := map[string]struct{}{
+		dataLibraries: {},
+		dataMessages:  {},
+	}
```

Comment:
if this is supposed to be the same as what groups allows could you please factor the message stub and `allowedCats` out into a common variable